test/ngMock/angular-mocksSpec.js

```diff
@@ -1857,6 +1857,65 @@ describe('ngMock', function() {
       });
     });
   });
+
+
+  describe('$componentController', function() {
+    it('should instantiate a simple controller defined inline in a component', function() {
+      function TestController($scope, a, b) {
+        this.$scope = $scope;
+        this.a = a;
+        this.b = b;
+      }
+      module(function($compileProvider) {
+        $compileProvider.component('test', {
+          controller: TestController
+        });
+      });
+      inject(function($componentController, $rootScope) {
+        var $scope = {};
+        var ctrl = $componentController('test', {$scope: $scope, a: 'A', b: 'B'}, { x: 'X', y: 'Y' });
+        expect(ctrl).toEqual({$scope: $scope, a: 'A', b: 'B', x: 'X', y: 'Y' });
```

Comment:
JSCS does not check for this. We'd need either the [disallowSpacesInsideObjectBrackets][1] rule or the [requireSpacesInsideObjectBrackets][2] rule, which we don't have.

I prefer disallowing spaces (which is more consistent with our disallowing spaces inside square brackets and parenthesis). Should I add a rule for object brackets ?

[1]: http://jscs.info/rule/disallowSpacesInsideObjectBrackets
[2]: http://jscs.info/rule/requireSpacesInsideObjectBrackets